pop-0001.txt

```diff
@@ -0,0 +1,623 @@
+   for PyLadies. It may also describe an interoperability standard that will
+   be supported outside the standard library for current PyLadies versions
+   before a subsequent POP adds standard library support in a future
+   version.
+
+2. An **Informational** POP describes a PyLadies design issue, or
+   provides general guidelines or information to the PyLadies community,
+   but does not propose a new feature.  Informational POPs do not
+   necessarily represent a PyLadies community consensus or
+   recommendation, so users and implementers are free to ignore
+   Informational POPs or follow their advice.
+
+3. A **Process** POP describes a process surrounding PyLadies, or
+   proposes a change to (or an event in) a process.  Process POPs are
+   like Standards Track POPs but apply to areas other than the PyLadies
+   language itself.  They may propose an implementation, but not to
+   PyLadies's codebase; they often require community consensus; unlike
+   Informational POPs, they are more than recommendations, and users
+   are typically not free to ignore them.  Examples include
+   procedures, guidelines, changes to the decision-making process, and
+   changes to the tools or environment used in PyLadies development.
+   Any meta-POP is also considered a Process POP.
+
+
+POP Workflow
+============
+
+
+PyLadies's BDFL
+-------------
+
+There are several reference in this POP to the "BDFL". This acronym stands
+for "Benevolent Dictator for Life" and presently refers to Lynn Root, the
+current PyLadies liason to the PSF.
+
+
+POP Editors
+-----------
+
+The POP editors are individuals responsible for managing the administrative
+and editorial aspects of the POP workflow (e.g. assigning POP numbers and
+changing their status).  See `POP Editor Responsibilities & Workflow`_ for
+details.  The current editors are:
+
+* Selena Deckelman
+* Jackie Kaziel
+* Esther Nam
+* Audrey Roy
+* Carol Willing
+
+POP editorship is by invitation of the current editors.  The address
+<POPs@PyLadies.org> is a mailing list for contacting the POP editors.  All
+email related to POP administration (such as requesting a POP number
+or providing an updated version of a POP for posting) should be sent to
+this address (no cross-posting please).
+
+
+Submitting a POP
+----------------
+
+The POP process begins with a new idea for PyLadies.  It is highly
+recommended that a single POP contain a single key proposal or new
+idea. Small enhancements or patches often don't need
+a POP and can be injected into the PyLadies development workflow with a
+patch submission to the PyLadies `issue tracker`_. The more focused the
+POP, the more successful it tends to be.  The POP editors reserve the
+right to reject POP proposals if they appear too unfocused or too
+broad.  If in doubt, split your POP into several well-focused ones.
+
+Each POP must have a champion -- someone who writes the POP using the
+style and format described below, shepherds the discussions in the
```

Comment:
Actually proposing a POP could be as simple as a Pull Request against this repo, right?  Then x-post the link to the relevant list(s) for solicitation of comments.